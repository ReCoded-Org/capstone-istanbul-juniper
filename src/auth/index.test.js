import React from "react";
import renderer from "react-test-renderer";
import AuthProvider from "./authContext";

it("AuthProvider renders correctly", () => {
  const tree = renderer.create(<AuthProvider />).toJSON();
  expect(tree).toMatchSnapshot();
});
