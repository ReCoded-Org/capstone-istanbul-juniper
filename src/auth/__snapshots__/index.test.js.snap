// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthProvider renders correctly 1`] = `null`;
